CHANGELOG.md

```diff
@@ -5,10 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 2.9.0
 
 ### Added
 
+- Upgrade infrastructure-bundle to 2.8.1
+    - See https://github.com/newrelic/infrastructure-bundle/releases/tag/2.8.1 for more details about the upgraded integrations in this release of the infrastructure-bundle
+
```

Comment:
Could we keep the `# Unreleased` header with the dividing bar `---` and put 2.9.0 below?

Reply:
sure!